### PR TITLE
V10: Verifying Forms and Deploy

### PR DESCRIPTION
**Pull request by sofietoft**

### CODE REVIEW

File: Add-ons/UmbracoForms/Installation/Version-Specific.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 versionFrom: 7.0.0
-needsV9Update: "true"
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Installation/Version-Specific.md:3]: No versionTo tag?

File: Add-ons/UmbracoForms/Installation/ManualUpgrade-v8.md [hunk 2]
```diff
@@ -8,9 +8,11 @@ needsV9Update: "true"
 # Manually upgrading forms
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Installation/ManualUpgrade-v8.md:8]: No versionTo tag?

File: Add-ons/UmbracoForms/Developer/IFileSystem/index.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
 ---
 versionFrom: 7.0.0
-needsV8Update: "true"
 versionRemoved : 9.0.0
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Developer/IFileSystem/index.md:3]: versionTo: 8.0.0?

**Review by eshanrnh — APPROVED** [review]
Looks great 👏 Just a couple of comments/queries. 🙂

File: Add-ons/UmbracoForms/Developer/IFileSystem/index.md (hunk 3 above)
Comment (sofietoft) [reply Add-ons/UmbracoForms/Developer/IFileSystem/index.md:3]: See the note I made in lines 8-10.

File: Add-ons/UmbracoForms/Installation/ManualUpgrade-v8.md (hunk 2 above)
Comment (sofietoft) [reply Add-ons/UmbracoForms/Installation/ManualUpgrade-v8.md:7]: We usually add the `versionTo` tag, when there's an article for the later versions, right?

image

File: Add-ons/UmbracoForms/Developer/IFileSystem/index.md (hunk 3 above)
Comment (sofietoft) [reply Add-ons/UmbracoForms/Developer/IFileSystem/index.md]: Added the `versionTo: 8.0.0`.
That should have been added in the first place - good catch! 💪

**Review by eshanrnh — APPROVED** [review]
All good 💪 Merging in 🙂

**Pull request merged**